### ISSUE TITLE
two list select component?

### ISSUE THREAD

**Issue by DeadlyEmu** [Megabit/Blazorise#3311]
I am after adding two lists select system to a component where you move selected items from one list to another to enable them for a group/ system. only way I can see this being done currently is with two select components in multiple set to true and then buttons to control between them. if this is the best way I will go down that route but wondering if it would be a standalone component of blazorise in future. 

it would be good to have a single component I can attach the two lists to. available options and selected options. maybe even a single list with a selected attribute in the class. 

screen shot of another system 
![image](https://user-images.githubusercontent.com/6854643/149181660-722937bd-9834-4d1f-9953-6bee17c937a4.png)

code pen for the above [here](https://codepen.io/esausilva/pen/NYObRr)

**Comment by stsrki** [comment]
You can also try with the ListView component. Seems like a better choice.

https://blazorise.com/docs/extensions/list-view

**Comment by DeadlyEmu** [comment]
> You can also try with the ListView component. Seems like a better choice.
> 
> https://blazorise.com/docs/extensions/list-view

this does not allow multiple selection. 
for example: like you have with a select view.

**Comment by stsrki** [comment]
Yes, it seems it doesn't support multiple selections. @David-Moreira we might add this for 1.1?

Regarding your "problem". For now, the `Select` component seems like a good alternative. You might even try with the DataGrid component. Llthough it might be overkill.

**Comment by DeadlyEmu** [comment]
I will build with the select then at the moment but I do prefer the format of the list view so if we can get the multi select option :)
